functions/Functions.fs

```diff
@@ -226,6 +226,29 @@ module Functions =
         match u.ParentId with 
         | Some(id) -> validateUnitExists' id u
         | None -> ok u |> async.Return
+
+    let testForCircularDependency (u:Unit) (child:Unit option) =    
+        match (child) with
+        | Some(c) -> 
+            printfn "Found child %A" c
```

Comment:
Should remove these `printfn` statements?

Reply:
Removed.